software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -107,10 +107,10 @@
 
     // NB these two needed for clients to access
     @SetFromFlag("riakWebPort")
-    PortAttributeSensorAndConfigKey RIAK_WEB_PORT = new PortAttributeSensorAndConfigKey("riak.webPort", "Riak Web Port", "8098+");
+    PortAttributeSensorAndConfigKey RIAK_WEB_PORT = ConfigKeys.newPortSensorAndConfigKey("riak.web.port", "Riak Web Port", "8098+");
```

Comment:
Changing this name breaks backwards compatibility. If someone has in their blueprint `riak.webPort: 1234` then that config option will now be ignored. Same for `riak.pbPort: 1235`.